apiextensions/storageversion/cmd/migrate/main.go

```diff
@@ -62,6 +71,10 @@ func main() {
 	for _, gr := range grs {
 		logger.Info("Migrating group resource ", gr)
 		if err := migrator.Migrate(ctx, gr); err != nil {
+			if ignoreNotFound && apierrs.IsNotFound(err) {
+				logger.Infof("Ignoring resource migration - unable to fetch crd %s - %w", gr, err)
```

Comment:
`%w` only works for wrapping errors but here we are logging so we should use `%s`.

Otherwise the text is garbled in the log output - ie. https://go.dev/play/p/jyVUTGUxMJU

```suggestion
				logger.Infof("Ignoring resource migration - unable to fetch crd %s - %s", gr, err)
```